packages/api/src/controllers/stripe.js

```diff
@@ -0,0 +1,141 @@
+import Router from 'express/lib/router'
+import { db } from '../store'
+const app = Router()
+const stripe = require('stripe')(process.env.LP_STRIPE_SECRET_KEY)
+const endpointSecret = process.env.LP_STRIPE_WEBHOOK_SECRET
+import { products } from '../config'
+
+// Webhook handler for asynchronous events.
+app.post('/webhook', async (req, res) => {
+  const sig = req.headers['stripe-signature']
+
+  let event
+
+  try {
+    event = stripe.webhooks.constructEvent(req.body, sig, endpointSecret)
+  } catch (err) {
+    res.status(400).send(`Webhook Error: ${err.message}`)
+  }
+  if (event.type === 'invoice.created') {
+    let invoice = event.data.object
+
+    if (invoice.status === 'draft') {
+      const { data: userIds } = await req.store.query({
+        kind: 'user',
+        query: { stripeCustomerId: invoice.customer },
+      })
+      const user = await req.store.get(`user/${userIds[0]}`, false)
```

Comment:
Cleaner and safer b/c no replica:
```suggestion
      const [user] = db.user.find({ stripeCustomerId: invoice.customer }, {useReplica: false})
```

Reply:
@iameli nice. Does `db.user.find()` return users inside an array?

Reply:
In other words do I need to access the user like so:
```
const [users] = db.user.find({ stripeCustomerId: invoice.customer }, {useReplica: false})
const user = user[0]
```

Reply:
Fixed in [04ff25](https://github.com/livepeer/livepeer-com/pull/25/commits/04ff2537a397629316e3ccef0ffba6a97665ad0a)